### PR TITLE
Add product property to ApplicationInstance

### PR DESCRIPTION
**Pull request by marcospri**

### CODE REVIEW

File: lms/models/application_instance.py [hunk 1]
```diff
@@ -18,6 +19,16 @@
...
 
+    class Product(str, Enum):
+        BLACKBOARD = "BlackboardLearn"
+        CANVAS = "canvas"
+        MOODLE = "moodle"
+        D2L = "desire2learn"
+        BLACKBAUD = "BlackbaudK12"
+        ILIAS = "ilias"
+        SAKAI = "sakai"
+        UNKNOWN = "unknown"
```
Comment (marcospri) [inline lms/models/application_instance.py:30]: All values from the DB

File: lms/models/application_instance.py [hunk 2]
```diff
@@ -18,6 +19,16 @@
 class ApplicationInstance(BASE):
     """Class to represent a single lms install."""
 
+    class Product(str, Enum):
```
Comment (marcospri) [inline lms/models/application_instance.py:22]: I prefer this to be nested as you can do check against the Enum without needing to import anything new

Comment (marcospri) [reply lms/models/application_instance.py:22]: (str,Enum) makes this a str too so: 

Product("canvas") == Product.CANVAS == "canvas"

File: lms/models/application_instance.py (hunk 1 above)
Comment (seanh) [reply lms/models/application_instance.py:30]: Can we remove Ilias? I have no idea what that is! And there's only one instance of it in the DB.

There's also "schoology" (seven instances in the DB). We may want to add that

**Review by seanh — APPROVED** [review]

**Pull request merged**